bodhi/client/__init__.py

```diff
@@ -244,7 +274,8 @@ def composes():
 @staging_option
 @click.option('-v', '--verbose', is_flag=True, default=False, help='Display more information.')
 @url_option
-def list_composes(url, staging, verbose):
+@debug_option
+def list_composes(url, staging, verbose, debug):
```

Comment:
Note that we still document parameters in the "developer docs" section below, here and elsewhere in the file.

Reply:
Will add